torchtext/experimental/vectors.py

```diff
@@ -0,0 +1,43 @@
+import csv
+import torch
+
+
+def vectors_from_csv_file(file_like_object, unk_tensor=None):
+    r"""Create a Vectors object from a csv file like object.
+
+    Args:
+        file_like_object (FileObject): a file like object to read data from.
+        unk_tensor (int): a 1d tensors representing the vector associated with an unknown token
+    Returns:
+        Vectors: a Vectors object.
+    """
+    readCSV = csv.reader(file_like_object, delimiter=',')
+
+    tokens = []
+    vectors = []
+    for row in readCSV:
+        tokens.append(row[0])
+        vectors.append(torch.tensor([float(c) for c in row[1].split()]))
+
+    return Vectors(tokens, vectors, unk_tensor=unk_tensor)
+
+
+class Vectors(object):
+    r"""Creates a vectors object which maps tokens to vectors.
+
+    Arguments:
+        tokens (List[str]): a list of tokens.
+        vectors (List[torch.Tensor]): a list of 1d tensors representing the vector associated with each token.
```

Comment:
Should we also explain our warnings in the doc here (e.g. tokens and vectors are required to have same length; or empty list will happen ...)?

Reply:
I will add a section for `Raises:`